frontend/src/components/ListingsFeed.js

```diff
@@ -1,19 +1,98 @@
-            <p>{listing.organisationName}</p>
-            <p>{listing.description}</p>
-            <p>{listing.price}</p>
-            <p>{listing.location}</p>
-            <p>{listing.contact}</p>
-            <Link to={`/listings/${listing._id}`}>View Listing</Link>
+        <div className="listings-feed">
+            <div className="listings-feed__header">
+                <h2 className="listings-feed__title">Listings</h2>
+                <NewRequestForm />
+                <h2>Filter by requirement</h2>
+                <div className="listings-feed__filter">
+                    {listing.map((listing) => (
+                        <Link to={`/listings/${listing.requirement}`}>
+                            <button className="listings-feed__filter-button">
+                                {listing.requirement}
+                            </button>
+                        </Link>
+                    ))}
+                </div>
+            <div className="listings-feed__listings">
+                {listing.map((listing) => (
+                    <div className="listing" key={listing._id}>
+                        <div className="listing__header">
+                            <h3 className="listing__title">{listing.title}</h3>
+                            <p className="listing__date">
+                                {formatDistanceToNow(new Date(listing.createdAt), {
+                                    addSuffix: true,
+                                })}
+                            </p>
+                        </div>
+                        <div className="listing__body">
+                            <p className="listing__description">
+                                {listing.description}
+                            </p>
+                            <p className="listing__address">
+                                {listing.firstLine}
+                                <br />
+                                {listing.city}
+                                <br />
+                                {listing.postcode}
+                            </p>
+                            <p className="listing__needed-by">
+                                Needed by: {formattedDate}
+                            </p>
+                        </div>
+                        <div className="listing__footer">
+                            <Link
+                                to={`/listings/${listing._id}`}
+                                className="listing__link"
+                            >
+                                View listing
+                            </Link>
+                            <button
+                                className="listing__edit"
+                                onClick={() => handleEdit(listing)}
+                            >
+                                Edit
+                            </button>
+                        </div>
+                    </div>  
+                ))}
+            </div>
```

Comment:
Will remove this, forgot I was playing around with something and this doesn't need to be on here.